OWNERS_ALIASES

```diff
@@ -12,6 +12,12 @@ aliases:
     - elikatsis
     - kimwnasptd
     - thesuperzapper
+  wg-deployment-leads:
+    - Jeffwan
+    - animeshsingh
+    - swiftdiaries
```

Comment:
Should @swiftdiaries be removed from the list?

Reply:
resolved